sdl-debezium/src/test/scala/io/smartdatalake/workflow/dataobject/DebeziumCdcDataObjectMySqlIT.scala

```diff
@@ -0,0 +1,177 @@
+/*
+ * Smart Data Lake - Build your data lake the smart way.
+ *
+ * Copyright © 2019-2024 ELCA Informatique SA (<https://www.elca.ch>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package io.smartdatalake.workflow.dataobject
+
+import io.smartdatalake.app.{DefaultSmartDataLakeBuilder, SmartDataLakeBuilderConfig}
+import io.smartdatalake.config.{ConfigToolbox, InstanceRegistry}
+import io.smartdatalake.definitions.Environment
+import io.smartdatalake.testutils.TestUtil
+import io.smartdatalake.util.hdfs.HdfsUtil
+import io.smartdatalake.util.misc.SmartDataLakeLogger
+import io.smartdatalake.util.secrets.StringOrSecret
+import io.smartdatalake.workflow.action.{ActionMetadata, CopyAction}
+import io.smartdatalake.workflow.connection.authMode.BasicAuthMode
+import io.smartdatalake.workflow.connection.jdbc.JdbcTableConnection
+import io.smartdatalake.workflow.connection.DebeziumConnection
+import org.apache.hadoop.fs.{FileSystem, Path}
+import org.apache.spark.sql.functions.{col, lit}
+
+import java.nio.file.Files
+
+object DebeziumCdcDataObjectMySqlIT extends App with SmartDataLakeLogger {
```

Comment:
I get the following exception when executing this on demo Database with empty tables:
```
Exception in thread "main" io.smartdatalake.util.dag.TaskFailedException: Task copyAction1 failed. Root cause is 'NoSuchElementException: head of empty list'
Caused by: java.util.NoSuchElementException: head of empty list
        at scala.collection.immutable.Nil$.head(List.scala:469)
        at scala.collection.immutable.Nil$.head(List.scala:466)
        at io.smartdatalake.workflow.dataobject.DebeziumEventConverter$.convert(DebeziumCdcDataObject.scala:206)
```